documentai/snippets/handle_response_sample_test.py

```diff
@@ -42,6 +42,30 @@ def test_process_document_ocr(capsys):
     assert "en" in out
     assert "FakeDoc" in out
 
+    # Document Quality
+    assert "Quality score" in out
+
+    # Font Detection
+    assert "Font Size" in out
+    assert "Handwritten" in out
+
+    # Checkbox Detection
```

Comment:
suggestion: As this is another call to the service I suggest moving it out to a separate test.